CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [2.0.0](https://github.com/deshaw/jupyterlab-limit-output/compare/v1.0.1...v2.0.0) (2023-04-04)
+
+### Changed
+
+- Ported to JupyterLab 4.
```

Comment:
Let's mark this as breaking (e.g. similar to https://github.com/deshaw/jupyterlab-execute-time/blob/master/CHANGELOG.md#200-2021-01-19)